handler/implementations/sysDevicesVirtual.go

```diff
@@ -104,15 +109,19 @@ func (h *SysDevicesVirtual) Lookup(
 		return info, nil
 	}
 
-	// XXX: for non emulated resources under /sys/devices/virtual, we should
-	// ideally request the passthrough handler to perform the lookup; however
-	// this slows down the lookup and causes sysbox containers with systemd
-	// inside to fail in hosts with kernel < 5.19 (i.e., systemd takes too long
-	// to boot because for some reason it's doing lots of lookups of
-	// /sys/virtual/devices/block/loopX devices, causing it to timeout).  Instead
-	// we do the lookup at host level, which is perfectly fine except since we
-	// don't know of any files under /sys/virtual/devices which Linux exposes at
-	// host level but not inside containers.
+	// For non emulated resources under /sys/devices/virtual, we should
```

Comment:
We need to extend this to other file ops -- at least read() -- otherwise, some apps will continue to fail after discovering the existence of these `*/net` nodes.

Reply:
Yep, good point, I missed that. I've added that now, thanks!